src/sfizz/FilePool.h

```diff
@@ -68,12 +68,13 @@ struct FilePromise
             return AudioSpan<const float>(fileData)
                 .subspan(sfz::config::excessFileFrames,
                     fileData.getNumFrames() - 2 * sfz::config::excessFileFrames);
-        else if (availableFrames > preloadedData->getNumFrames())
-            return AudioSpan<const float>(fileData).subspan(sfz::config::excessFileFrames, availableFrames);
+        else if (availableFrames > preloadedData->getNumFrames() + sfz::config::excessFileFrames)
+            return AudioSpan<const float>(fileData)
+                .subspan(sfz::config::excessFileFrames, availableFrames - sfz::config::excessFileFrames);
```

Comment:
Is the use of `availableFrames` correct here, considering that this value does not account for any padding?

I would think this 1st and 3rd line to be correct instead:
```
        else if (availableFrames > preloadedData->getNumFrames() - 2 * sfz::config::excessFileFrames)
```

```
                .subspan(sfz::config::excessFileFrames, availableFrames);
```

Reply:
Actually it does because the streaming function is passed a span, and not the raw buffer, so it should be the real number of available frames. 

The check actually should probably be `availableFrames > preloadedData->getNumFrames() - sfz::config::excessFileFrames`. The reasoning is that you want that the reported number of available frames is `excessFileFrames` less than the actual number of available frames, so that the interpolator will not read beyond the actual end. This was the reasoning of the change I made, but I did not take into accout that `preloadedData->getNumFrames()` has an excess of `2 * sfz::config::excessFileFrames` frames already...

Reply:
I resolved the padding problem at AudioBuffer level instead of FilePool.
This should remove all `excessFileFrames` uglyness everywhere, at keep it as simple as before.